R-proj/src/sample_points.cpp

```diff
@@ -101,7 +107,8 @@ Rcpp::NumericMatrix sample_points(Rcpp::Nullable<Rcpp::Reference> P = R_NilValue
 
     int type, dim, numpoints;
     NT radius = 1.0, delta = -1.0;
-    bool set_mean_point = false, cdhr = true, rdhr = false, ball_walk = false, gaussian = false;
+    bool set_mean_point = false, cdhr = false, rdhr = false, ball_walk = false, gaussian = false,
```

Comment:
Why having all those `bool` vars, one per walk type and not just one `enum` that takes the value related to the walk type, e.g. `enum Walk { RDHnR, CDHnR, Ball, Billiard, Vaidya, John, Dikin, ... };`